bundles/org.openhab.ui.basic/web-src/smarthome.js

```diff
@@ -1337,8 +1339,37 @@
 				icon.addEventListener("load", _t.convertToInlineSVG);
 				icon.addEventListener("error", _t.replaceImageWithNone);
 			}
+
+			if (maxButtonWidth < button.offsetWidth) {
+				maxButtonWidth = button.offsetWidth;
+			}
 		});
 
+		if (_t.buttons.length > 1 && _t.parentNode.classList.contains(o.buttonsMultilineClass)) {
```

Comment:
isn't your minimize code also expected when there is only one button ?
As I understood, this will provide more place for label (and value).

Reply:
Maybe not because the height will never change if we have only one button ?

Reply:
hmm it has been a while since I worked on this, but from my vague memory, there _was_ a reason why it was only applied to > 1 buttons. Oh yes, it came to me just now... 1 button cannot "wrap" around, so it's not possible to detect the breakpoint in the manner it's done in the loop above.

Reply:
Perhaps I should add a comment to remind myself (and explain to others) about this in the future?

Reply:
But buttons.style.width will already be optimized in that case, no ?
I will include in my tests the case of only one button to be sure that the label is not smaller than it should be.

Reply:
> But buttons.style.width will already be optimized in that case, no ?

Not through js, but through the various tweaks (statically) made on the scss.